### PR TITLE
docs: revise grammatical errors and typos of MD

### LINKED ISSUE #673: [Docs] a broken link in CONTRIBUTING
### Description

Due to the change of files in the repository, the link in the contributing.md is invalid. I will fix this small problem immediately.

### PR DESCRIPTION
**Pull request by andyhuang18**
## Brief Information

This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):

- [ ] build
- [ ] ci
- [x] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [ ] refactor
- [ ] test

Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):

- resolve #673 

## Details

- Clicking on the link now redirects correctly.
- Modified the grammatical errors and typos in the md.

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

### CODE REVIEW

**Comment by andyhuang18** [comment]
Hi~ @lhbvvvvv . Can you help to reveiw my PR? It's a quick work. Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -179,5 +179,5 @@ Please sign our [Contributor License Agreement (CLA)][cla] before sending pull r
 [repo]: https://github.com/hypertrons/hypertrons-crx
 [angular-commit-message-format]: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines
 [pr-template]: ./.github/pull_request_template.md
-[issue-template]: ./.github/issue_template.md
+[issue-template]: https://github.com/hypertrons/hypertrons-crx/issues/673
```
Comment (tyn1998) [inline CONTRIBUTING.md:182]: Hi @andyhuang18, the link for issue-template should be this one: https://github.com/hypertrons/hypertrons-crx/tree/master/.github/ISSUE_TEMPLATE

and another suggestion:

image

**Review by tyn1998 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (andyhuang18) [reply CONTRIBUTING.md:182]: Copy that

**Review by tyn1998 — APPROVED** [review]
LGTM, thanks for your contribution! @andyhuang18 

/approve

**Pull request merged**